cogs/AutoResponse.py

```diff
@@ -43,34 +43,41 @@ async def enable(self, interaction: discord.Interaction, button: discord.ui.Butt
 class AutoResponse(commands.Cog):
     def __init__(self, bot :commands.Bot) -> None:
         self.bot = bot
-
+        self.last_time = datetime.datetime.utcfromtimestamp(0)
+    
     @commands.Cog.listener()
     async def on_message(self, message):
         view = SimpleView()
-        data = util.JsonHandler.load_users()
+        users = util.JsonHandler.load_users()
+        channel = util.JsonHandler.load_channels()
+        time_diff = (datetime.datetime.utcnow() - self.last_time).total_seconds()
 
-        if replycheck() == True:
-            if str(message.author.id) in data:
-                if str(message.author.id) == False:
-                    return
+        if time_diff < config["cooldowntime"]:
+            return
```

Comment:
a line like `print(f"Tried to send message while on cooldown! Didn't send message!")` would be nice

apart from that, looks good to merge